linera-views/src/unit_tests/views.rs

```diff
@@ -382,3 +377,160 @@ async fn test_clearing_of_cached_stored_hash() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+/// Check if a [`ReentrantCollectionView`] doesn't have pending changes after loading its
```

Comment:
```suggestion
/// Checks if a [`ReentrantCollectionView`] doesn't have pending changes after loading its
```

(Also below.)

Or maybe `Checks that` or `Verifies that` or `Asserts that` would be better, to make it clear that that's the expected outcome.